bundles/org.openhab.binding.freeathomesystem/README.md

```diff
@@ -0,0 +1,56 @@
+# FreeAtHomeSystem Binding
```

Comment:
Did you commit the wrong readme? It consists only of placeholders.

The readme shouldn't contain any UI specific things like screenshots. So, it will stay up-to-date if the UI changes.

Reply:
Fix is done, the readme.md file is up-to-date. No images are used.